src/rpc/rawtransaction.cpp

```diff
@@ -574,7 +564,7 @@ static RPCHelpMan decodescript()
                  {
                      {RPCResult::Type::STR, "asm", "String representation of the script public key"},
                      {RPCResult::Type::STR_HEX, "hex", "Hex string of the script public key"},
-                     {RPCResult::Type::STR, "type", "The type of the script public key (e.g. witness_v0_keyhash or witness_v0_scripthash)"},
+                     {RPCResult::Type::STR, "type", "The type of the script public key (one of: " + GetAllOutputTypes() + ")"},
```

Comment:
pretty sure this is wrong. Only v0 can be wrapped, no?

Reply:
Reverted. Thanks.